client/landing/gutenboarding/utils.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * ***Do not reproduce this pattern for Gutenboarding work.***
+ *
+ * FIXME: Replace this with another pattern:
+ *   - Side-effect (control) of dispatched action?
+ */
+
+/**
+ * Internal dependencies
+ */
+import { SiteVertical } from './stores/onboard/types';
+import { DomainName } from '@automattic/data-stores/types/domain-suggestions';
+
+/**
+ * ⚠️😱 Calypso dependencies 😱⚠️
+ */
+import wpcom from '../../lib/wp';
+import { untrailingslashit } from '../../lib/route';
+import { urlToSlug } from '../../lib/url';
+
+interface CreateSite {
+	siteTitle?: string;
+	siteUrl?: DomainName;
+	theme?: string;
+	siteVertical: SiteVertical | undefined;
+}
+
+export function createSite( { siteTitle, siteUrl, theme, siteVertical }: CreateSite ) {
+	const newSiteParams = {
```

Comment:
@Automattic/cylon is there a site param that can be used to launch an FSE site easily in API query like this?

Reply:
👋 

> a site param that can be used to launch an FSE site easily in API query like this

Do you mean 'make the site public by default'? 

Site launch uses a separate endpoint: `/sites/${ siteIdOrSlug }/launch`

Or you could try just turning off privacy by default and setting `public: 1`

Reply:
Nope, we want to create private site here so that we can launch it via "launch" button.

I meant that the site would have the FSE experiment turned on so that we get little feel how it would be to end up with preview of the full site (instead of just homepage content) in this flow.

Reply:
Gotcha. 

We also have a task for that :) 

And it seems like you've just answered my question in the task comments :

https://github.com/Automattic/wp-calypso/issues/38798#issuecomment-579030076

Thanks! 👍 

At present, FSE is enabled if `'test-fse' === $site_creation_flow;`. 

So for now, you could set `site_creation_flow` to equal `'test-fse'` in the request object for testing, but we'll need to elaborate on that condition in the new sites class to either accept something like `'gutenboarding'` as the `$site_creation_flow` or maybe a more generic `full-site-editing: 1`.

Reply:
Thanks!

Just learned via Slack the bit about flow name activating it.

Slack convo for the record p1580327754282800-slack-cylon